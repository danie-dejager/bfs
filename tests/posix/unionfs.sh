[[ "$UNAME" == *BSD* ]] || skip
clean_scratch
"$XTOUCH" -p scratch/{lower/{foo,bar,baz},upper/{bar,baz/qux}}
bfs_sudo mount -t unionfs -o below scratch/{lower,upper} || skip
trap "bfs_sudo umount scratch/upper" EXIT
bfs_diff scratch
