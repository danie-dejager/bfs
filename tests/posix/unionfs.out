scratch
scratch/lower
scratch/lower/bar
scratch/lower/baz
scratch/lower/foo
scratch/upper
scratch/upper/bar
scratch/upper/baz
scratch/upper/baz/qux
scratch/upper/foo
